swiftsimio/visualisation/sphviewer.py

```diff
@@ -162,18 +189,30 @@ def get_camera(
         extent: Union[None, List[float]] = None,
     ):
         """
-        Get the py-sphviewer camera object. This also sets it as self.camera that is used later.
-
-        Properties are:
-
-        + x, y, z: Cartesian co-ordinates of the object you're looking at
-        + r: Cartesian distance to the object
-        + t: ?????????? TODO
-        + p: ?????????? TODO
-        + zoom: ?????????? TODO
-        + roll: ?????????? TODO
-        + xsize, ysize: Pixel size of your output
-        + extent: Area to render between
+        Get the py-sphviewer camera object. 
+        
+        Parameters
+        ----------
+        x, y, z : float, optional
+            Cartesian co-ordinates of the object being viewed
+        r : float, optional
+            Cartesian distance to the object
+        t : float, optional
+            ?????????? TODO
+        p : float, optional
+            ?????????? TODO
+        zoom : float, optional
```

Comment:
Unfortunately I don't know what these parameters do now either - so can we say 'see py-sphviewer documentation`?